src/main/java/org/opentripplanner/routing/stoptimes/StopTimesHelper.java

```diff
@@ -76,49 +74,10 @@
 
     ServiceDate[] serviceDates = dates.toArray(new ServiceDate[dates.size()]);
 
-    // TODO The following logic could probably be encapsulated in the TimetableSnapshot
-    Collection<TripPattern> plannedPatterns = routingService.getPatternsForStop(stop, false);
-    Collection<TripPattern> realTimePatterns = routingService.getPatternsForStop(stop, true);
-
-    // Only include the planned patterns that are not also part of the collection of realtime
-    // patterns. These may have been cancelled with no replacement.
-    plannedPatterns.removeAll(realTimePatterns);
-
-    // Realtime patterns may have been cancelled and replaced. Do not include the patterns that
-    // have been replaced
-    Collection<TripPattern> replacedPatterns = realTimePatterns
-        .stream()
-        .map(TripPattern::getOriginalTripPattern)
-        .filter(Objects::nonNull)
-        .collect(Collectors.toList());
-
-    realTimePatterns.removeAll(replacedPatterns);
-
-    /*
-     First, check all the TripPatterns that are planned, but not part of the realtime patterns. This
-     is so that we catch TripPatterns that have been cancelled by realtime updates.
-     */
-    for (TripPattern pattern : plannedPatterns) {
-      Queue<TripTimeOnDate> pq = listTripTimeShortsForPatternAtStop(
-          routingService,
-          timetableSnapshot,
-          stop,
-          pattern,
-          startTime,
-          timeRange,
-          numberOfDepartures,
-          arrivalDeparture,
-          includeCancelledTrips,
-          serviceDates
-      );
-
-      result.addAll(getStopTimesInPattern(pattern, pq));
-    }
+    // Fetch all patterns, including those from realtime sources
+    Collection<TripPattern> patterns = routingService.getPatternsForStop(stop, timetableSnapshot);
```

Comment:
I read the code down this call stack. The Javadoc for `Graph.index.getPatternsForStop(stop, snapshot)` is incorrect, I believe. If you agree, would you be so nice as to correct it?

Reply:
Fixed the parameter name, is there anything else?